### PR TITLE
Fix fee calculation

### PR DESCRIPTION
**Pull request by notlesh**
Fixes client-side fee calculation, similar to what was done here: https://github.com/PureStake/moonbeam/pull/2254. Notably, this queries the new `api.call.transactionPayment.weightToFee` and `...lengthToFee` which use the underlying functions configured in the runtime for this pallet.

### CODE REVIEW

File: src/utils/monitoring.ts [hunk 1]
```diff
@@ -206,10 +206,10 @@ export const getAccountFromNimbusKey = async (
     !authorMappingCache[nmbsKey] ||
     authorMappingCache[nmbsKey].lastUpdate < Date.now() - 3600 * 1000
   ) {
-    const mappingData = (await api.query.authorMapping.mappingWithDeposit(nmbsKey)) as Option<any>;
+    const mappingData = (await api.query.authorMapping.mapping(nmbsKey)) as Option<any>;
```
Comment (crystalin) [inline src/utils/monitoring.ts:209]: Where does this come from ? Did we change the storage item in our AuthorMapping pallet ?

Comment (notlesh) [reply src/utils/monitoring.ts:209]: Good question. I remember being confident about the change when I made it, but now I can't seem to figure that out again. It looks like it should still be `mappingWithDeposit`...

Comment (notlesh) [reply src/utils/monitoring.ts:209]: This seems wrong to me, so I made an issue to track it. Thanks for questioning it.

Comment (crystalin) [reply src/utils/monitoring.ts:209]: No wories. I pushed a fix in main.

**Pull request merged**